lock-impl/src/main/java/com/palantir/lock/logger/LockStateYamlWriter.java

```diff
@@ -43,7 +44,8 @@
      * Creates a LockStateYamlWriter for the given file in append mode.
      */
     public static LockStateYamlWriter create(File file) throws IOException {
-        return new LockStateYamlWriter(new BufferedWriter(new FileWriter(file, true)));
+        return new LockStateYamlWriter(Files.newBufferedWriter(
+                file.toPath(), StandardCharsets.UTF_8, StandardOpenOption.CREATE, StandardOpenOption.APPEND));
```

Comment:
This is probably the only change we want to sanity check

Reply:
Is this file ever read? Can you write a test and see if this works locally?

Reply:
I have literally no idea. Could do

Reply:
I believe `LockServiceTest` tests this code path